recipes/samtools/0.1.12/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+sed -i.bak 's/-lcurses/-lncurses/' Makefile
```

Comment:
OSX `sed` does not have `-i` ... not sure why this worked at all?

Reply:
Never mind. I overlooked that it just **requires** the backup extension

Reply:
It does have `-i`, but it forces the creation of the backup file, see https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/sed.1.html

I think this is correct.